packages/x-data-grid/src/hooks/features/virtualization/useGridVirtualization.tsx

```diff
@@ -22,7 +22,7 @@ export const EMPTY_RENDER_CONTEXT = {
 
 export const virtualizationStateInitializer: GridStateInitializer<RootProps> = (state, props) => {
   const virtualization = {
-    enabled: !props.disableVirtualization,
+    enabled: !props.disableVirtualization && !props.autoHeight,
```

Comment:
This disables row & column virtualization, is this the desired behavior?

Reply:
No it's not, I misunderstood because there is a separate `enabledForColumns` property. Will fix.

Reply:
I'm confused by the default state of `enabledForColumns` here. Why is it `true` regardless of the value of `props.disableVirtualization` if that prop disables virtualization for columns and rows?